app/controllers/user/populi_transfer_controller.rb

```diff
@@ -5,8 +5,8 @@ def new
 
   def create
     @attendance = Attendance.find(params[:attendance_id])
-    @attendance.transfer_to_populi!
-    flash[:success] = "Success! Student Attendances have been transferred to Populi. Please double check that the attendance in Populi is accurate."
+    PopuliTransferJob.perform_async(@attendance.id)
+    flash[:success] = "Transfering attendance to Populi. Please confirm in Populi that attendance is accurate."
```

Comment:
Would it be easy to provide a Populi link to the relevant bit of attendance being taken here? Wondering if after `Please confirm in Populi...` if it could also include the link to where one would go to check this. If it's not easy to infer or predict the Populi URL then it's probably not worth adding, but just a thought.

Reply:
Great idea! I just added that

Reply:
Good stuff! Well done 👏